gulpfile.js

```diff
@@ -38,16 +38,18 @@ function getBundleConfig() {
 
     if (RELEASE) {
         config.plugins = config.plugins.concat(
-            new webpack.optimize.UglifyJsPlugin(),
-            new webpack.optimize.OccurenceOrderPlugin()
+            new webpack.optimize.UglifyJsPlugin({
+                sourceMap: true
+            }),
+            new webpack.optimize.OccurrenceOrderPlugin()
         );
     } else {
         config.output.pathinfo = true;
     }
 
-    if (!RELEASE || DEBUG) {
-        config.devtool = '#inline-source-map';
-    }
+    config.devtool = (RELEASE)
+                        ? 'source-map'
```

Comment:
This form is not clearer for me, I always confused where does this `?` belong to. I was able to change it to one-liner for this case.